bin/send-sync-status-metrics

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env node
+require('dotenv').config()
+
+const Sentry = require('@sentry/node')
+
+Sentry.init({
```

Comment:
I see this init copy/pasted in two different places. Not a blocker for this effort, but I think it'd be great for us to abstract this out.

Perhaps something like:

```node
const { Sentry, initializeSentry } = require("sentry"); # our file
initializeSentry();
```

That way the other areas only need to change to:

```node
const { Sentry } = require("sentry");
```

_I don't feel that attached to that suggestion if there's a better node-ism._

Reply:
This bugged me too. [Previously](https://github.com/integrations/jira/pull/239), we did something similar but it led to initializing the client multiple times, causing duplicate errors.

Your suggested approach makes initialization more obvious and seems more likely to be used appropriately! I'll make a follow-up PR for that.